app/javascript/views/usage/index.js

```diff
@@ -14,12 +14,15 @@ $(() => {
 
   // attach listener to separator select menu
   // on change look for "stat" elements and chnage their query param
-  document.getElementById('csv-field-sep').addEventListener('click', (e) => {
-    const statElems = document.getElementsByClassName('stat');
-    const newSep = 'sep='.concat(encodeURIComponent(e.target.value));
-    const changeStatFn = changeStatFnGen(newSep);
-    Array.from(statElems).forEach(changeStatFn);
-  });
+  const fieldSep = document.getElementById('csv-field-sep');
+  if (fieldSep !== null) {
```

Comment:
Lack of nil check was causing errors on non-usage pages

Reply:
thanks for adding that